repository/src/main/resources/alfresco/template-services-context.xml

```diff
@@ -171,4 +171,25 @@
             <value>urldecode</value>
         </property>
     </bean>
+
+    <bean id="testDocumentUrl" parent="baseTemplateImplementation" class="org.alfresco.repo.template.DocumentationURLMethod">

```

Comment:
The same case as in `repository/src/main/resources/alfresco/subsystems/email/OutboundSMTP/mail-template-services-context.xml`